examples/tutorials/vectors.py

```diff
@@ -0,0 +1,486 @@
+
+# Create a list of lists that include each vector information
+vectors = [[2, 3, 45, 4]]  # vectors structure: [[ 2  3 45  4]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# Using the functionality mentioned in the previous example,
+# multiple vectors can be plotted at the same time. Another
+# vector could be simply added to the 2D list or numpy
+# array object
+# and passed using ``data`` parameter.
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_1 = [2, 3, 45, 4]
+vector_2 = [7.5, 8.3, -120.5, 7.2]
+# Create a list of lists that include each vector information
+vectors = np.array([vector_1, vector_2])
+# vectors structure:
+# [[   2.     3.    45.     4. ]
+#  [   7.5    8.3 -120.5    7.2]]
+
+fig = pygmt.Figure()
+fig.plot(
+    region=[0, 10, 0, 10],
+    projection="X10c/10c",
+    frame="ag",
+    data=vectors,
+    style="v0.6c+e",
+    pen="2p",
+    color="red3",
+)
+fig.show()
+
+########################################################################################
+# In this example, cartesian vectors are plotted over a Mercator
+# projection of the continental US. The x values represent the
+# longitude and y values represent the latitude where the vector starts.
+#
+# This example also shows some of the styles a vector supports.
+# ``+ba`` specifies that the begining point of the vector ``+b``
+# should take the shape of a circle ``c``. Similarly, the end
+# point of the vector ``+e`` should have an arrow shape ``a``
+# (to draw a plain arrow, use ``A`` instead). Lastly, the ``+a``
+# specifies the angle of the vector head apex (30 degrees in
+# this example).
+#
+# More styling options can be found here
+# :doc:`Vector heads and tails </gallery/lines/vector_heads_tails>`.
+
+# create a plot with coast, Mercator projection (M) over the continental US
+fig = pygmt.Figure()
+fig.coast(
+    region=[-127, -64, 24, 53],
+    projection="M10c",
+    frame="ag",
+    borders=1,
+    shorelines="0.25p,black",
+    area_thresh=4000,
+    land="grey",
+    water="lightblue",
+)
+
+style = "v0.4c+bc+ea+a30"
+fig.plot(
+    x=-110,
+    y=40,
+    style=style,
+    pen="1p",
+    color="red3",
+    direction=[[-25], [3]],
+)
+
+# vector specifications structured as: [x_start, y_start, direction_degrees, length]
+vector_2 = [-82, 40.5, 138, 2.5]
+vector_3 = [-71.2, 45, -115.7, 4]
+# Create a list of lists that include each vector information
+vectors = [vector_2, vector_3]
+
+fig.plot(
+    data=vectors,
+    style=style,
+    pen="1p",
+    color="yellow",
+)
+
+fig.show()
+
+########################################################################################
+# Another example of plotting cartesian vectors over a coast plot. This time
```

Comment:
This example is similar to the first vector in the example above except that several vectors are passed to x, y, and directions. How about merging this example with the one above? So first a single vector, then several vectors using `x`, `y`, and `directions` parameters, and the last is vectors using `data` parameter.

Reply:
This example is OK to me. "merging this example with the one above" will make the example too long and complicated to follow.

Reply:
I agree with @seisman, we were trying to focus on one thing in each example, and merging many examples seemed to make it too long and confusing. Regardless, I'm fine with making the changes you suggested. So what would you like us to do?

Reply:
Let's keep them separated.